src/project/project.service.ts

```diff
@@ -135,4 +135,55 @@ export class ProjectService {
 
         return project.save()
     }
+
+    async addUserToProject(
+        projectId: string,
+        userEmail: string,
+        role: string,
+        requestorId: string
+    ) {
+        const isAdmin = await this.userService.isAdmin(requestorId)
```

Comment:
this is an id or email ? we need to define if we use emails or ids to identify users

Reply:
es id